### PR TITLE
Document connector setup for namenode HA

### PR DESCRIPTION
**Pull request by rschlussel**
Explicitly document configuration for namenode HA.  We've encountered a
lot of people having issues configuring Presto for HA clusters.

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -49,17 +49,16 @@ will create a catalog named ``sales`` using the configured connector.
...
 ^^^^^^^^^^^^^^^^^^
 
-Presto configures the HDFS client automatically for most setups and
-does not require any configuration files. In some rare cases, such
-as when using federated HDFS, it may be necessary to specify additional
-HDFS client options in order to access your HDFS cluster. To do so, add
-the ``hive.config.resources`` property to reference your HDFS config files:
+For basic setups, Presto configures the HDFS client automatically and
+does not require any configuration files. In some cases, such as when using
+federated HDFS or namenode high availability, it is necessary to specify
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/connector/hive.rst]: capitalize namenode as NameNode?

**Comment by cawallin** [comment]
looks good

File: presto-docs/src/main/sphinx/connector/hive.rst (hunk 1 above)
Comment (rschlussel) [inline presto-docs/src/main/sphinx/connector/hive.rst]: done

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -49,17 +49,16 @@ will create a catalog named ``sales`` using the configured connector.
...
+does not require any configuration files. In some cases, such as when using
+federated HDFS or NameNode high availability, it is necessary to specify
+additional HDFS client options in order to access your HDFS cluster. To do so,
+add the ``hive.config.resources`` property to reference your HDFS config files:
 
 .. code-block:: none
 
     hive.config.resources=/etc/hadoop/conf/core-site.xml,/etc/hadoop/conf/hdfs-site.xml
 
-Only specify additional configuration files if absolutely necessary.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst]: This line should stay. Adding additional config files could cause problems, so it is better not to do it unless necessary.

Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst]: We can make the language softer: "Only specify additional configuration files if necessary for your setup."

Comment (rschlussel) [inline presto-docs/src/main/sphinx/connector/hive.rst]: It doesn't seem to add anything that isn't included in the next line about only including the minimum set of required properties.  But I can add it back with the softer language.  

Also, are there particular properties that are known to cause issues?  That might be worth documenting as well.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**